### DISCUSSION TITLE
Did the context clause `codeActionMenuVisible` disappear for v1.74?

### DISCUSSION THREAD

**Discussion by ArturoDent** [microsoft/vscode-discussions#294] [Extension Development QnA] [answered]
[Sorry, not really extension-related.]

I can't find the context value `codeActionMenuVisible` in Stable v1.74 or Insiders v1.75 anymore...  Was it removed?  I don't see a related issue about a change.

**Comment by isidorn** [comment]
I just searched our source code and I could not find any trace of `"codeActionsMenuVisible"` context.
@mjbvz might know better.

**Comment by ArturoDent** [comment, marked as answer]
@isidorn Thanks for checking.  A change was made to the context name but it was just reverted back to the old name with v1.74.1.  See https://github.com/microsoft/vscode/pull/168622
